unit/suites/test_execute.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+import sys
+import unittest
+
+import tarantool
+from .lib.tarantool_server import TarantoolServer
+
+
+class TestSuite_Execute(unittest.TestCase):
+    table_prefix = 'dbapi20test_'  # If you need to specify a prefix for tables
```

Comment:
It seems it is from dbapi test and looks a bit strange here.

Reply:
I removed table_prefix as it redundant in this test suite